app/javascript/data/advancedFilterOptions.js

```diff
@@ -111,27 +99,16 @@ export const advancedFilterOptions = [
     options: ['Exact Match', 'Contains'],
   },
   {
-    name: 'email',
-    title: 'Primary Contact Email (Text)',
-    description: 'Monitoree primary contact email address',
-    type: 'search',
-  },
-  {
-    name: 'first-name',
-    title: 'Name (First) (Text)',
-    description: 'Monitoree first name',
-    type: 'search',
-  },
-  {
-    name: 'last-name',
-    title: 'Name (Last) (Text)',
-    description: 'Monitoree last name',
+    name: 'telephone',
```

Comment:
Can we keep this list in alphabetical order by name?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1179/commits/75cfbf52593139c2a00f0928ae7b4db44a649c23